pom.xml

```diff
@@ -891,7 +954,7 @@
                             <goal>analyze-only</goal>
                         </goals>
                         <configuration>
-                            <failOnWarning>true</failOnWarning>
```

Comment:
This plugin is an easy way to warn if any unused dependencies are included. Unused dependencies can bloat the jar size, but more importantly, nothing sucks more than debugging/maintaining/paying attention to stuff that you don't actually use. 

If this is throwing off false positives or something, I would recommend ignoring specific dependencies or groups of dependencies rather than turning this off across the broad https://maven.apache.org/plugins/maven-dependency-plugin/analyze-only-mojo.html#ignoredDependencies

Reply:
Reverted to True.

Reply:
consonance-arch:
    Added execution and configuration to the pom. It locally does not fail on warning.